rust_src/src/xfaces.rs

```diff
@@ -0,0 +1,25 @@
+use crate::{
+    lisp::defsubr,
+    remacs_sys::{clear_face_cache, windows_or_buffers_changed},
+};
+use remacs_macros::lisp_fn;
+use std::sync::atomic::{AtomicBool, Ordering};
+
+lazy_static! {
+    static ref FACE_CHANGE: AtomicBool = AtomicBool::new(false);
```

Comment:
Do not replicate the variables from C when porting. This leads to bugs where C sets a value and Rust does not see it.
Remove the original `FACE_CHANGE` and see how it goes.

Reply:
If I remove the variable from C then I'll need to export a getter and a setter from Rust and use it in the C code until everything is ported. Do I understand correctly?

Reply:
Sounds right. Which is why we usually use the C variables until all of the code which needs them is ported then port the variable last.

Reply:
Maybe I'm missing something, but I don't see the original `face_change` listed in the doc of `remacs_sys`. How can I access the original C variable from Rust?

Reply:
If the variable is only used in the one C file then you need to expose it in a header.

Reply:
or right a C function to set/get and expose them.

Reply:
It appears [here](https://github.com/r-darwish/remacs/blob/clear-face-cache/src/dispextern.h#L1813).  Is it unavailable in `emacs_sys` due to [this](https://github.com/r-darwish/remacs/blob/clear-face-cache/rust_src/remacs-bindings/src/main.rs#L258) blacklist?

Reply:
I added up an setter function because I don't want to alter existing C code. We can add the getter one it's needed.